pages/business-import-export.js

```diff
@@ -18,30 +18,35 @@ import {
   HintText
 } from "govuk-react";
 import PropTypes from "prop-types";
+import { withTranslation } from "../i18n";
 
 const ImportExportActivities = (props) => (
   <FsaLayout {...props}>
     <PostForm action={props.formAction} csrfToken={props.csrfToken}>
-      <BackButton {...props} />
+      <BackButton {...props} t={props.t} />
```

Comment:
Is this explicit setting of 't' actually required, since we are spreading all the props onto the component anyway?

Reply:
Nope, this was highlighted in SonarCloud also, think David is in the process of removing these now.

Reply:
Yeah that's right